HM Dashboard Views

hpainter2002 opened the pull request.

hpainter2002 committed:
Commit for sync

hpainter2002 committed:
HiringManager-View User Info US-IP

hpainter2002 committed:
Merge remote-tracking branch 'refs/remotes/DavidA94/master'

hpainter2002 committed:
Commit to switch to desktop

hpainter2002 committed:
Merge remote-tracking branch 'refs/remotes/DavidA94/master'

hpainter2002 committed:
Commit to switch to laptop.

hpainter2002 committed:
Switching to Desktop

hpainter2002 committed:
Switching to Laptop.
Changes: InterviewPage added

hpainter2002 committed:
Merge remote-tracking branch 'refs/remotes/DavidA94/master'

hpainter2002 committed:
Commit for merge with master.
Changes: Added arrows to collapse

hpainter2002 committed:
Switching to Laptop
Changes: Refactored HM Dashboard, Added Logo, Added 6 new sections to the dashboard.

hpainter2002 committed:
Switch to Desktop.
Changes: Fixed Login Claims Identity

hpainter2002 committed:
Sync with master/AES

DavidA94 commented on AES.Web/Authorization/ApplicantUserManager.cs:
```diff
@@ -66,6 +65,7 @@ public static ApplicantLoginModel GetUser()
         public static int GetUserID()
         {
             try {
+                var x = HttpContext.Current.GetOwinContext().Authentication.User.Claims;
```
That's where that went. Can you drop it please?

DavidA94 commented on AES.Web/Controllers/HiringManagerController.cs:
```diff
@@ -0,0 +1,393 @@
+
+            // ** Get the store ID from the database**********
+            //ApplicantInfoContract[] InterviewApplicants = appSvc.GetApplicantsAwaitingInterview(EmployeeUserManager.GetUser().StoreID);
+            ApplicantInfoContract[] InterviewApplicants = appSvc.GetApplicantsAwaitingInterview(1);
```
This shouldn't exist anymore. Not sure how it managed to build. You might need to update your service references.

DavidA94 commented on AES.Web/Controllers/HiringManagerController.cs:
```diff
@@ -0,0 +1,393 @@
+
+            // ** Get the store ID from the database**********
+            //ApplicantInfoContract[] InterviewApplicants = appSvc.GetApplicantsAwaitingInterview(EmployeeUserManager.GetUser().StoreID);
+            ApplicantInfoContract[] InterviewApplicants = appSvc.GetApplicantsAwaitingInterview(1);
```
Ditto... and on the ones below as well.

DavidA94 commented on AES.Web/Controllers/HiringManagerController.cs:
```diff
@@ -0,0 +1,393 @@
+            };
+        }
+
+        public FullApplicationModel ConvertAppContractToModel(ApplicationInfoContract app)
```
You've got a region below to put the converters into, and this one and a couple others are not inside that.

hpainter2002 committed:
Changes made for pull request.

hpainter2002 committed:
Switching to Laptop
Changes: All the interview sections added using a tabbed container. All the views for interview section added as well.
